### PR TITLE
Users index

### PR DESCRIPTION
**Pull request by timdunn22**

### CODE REVIEW

**Comment by timdunn22** [comment]
Closes #162

**Comment by jamiefolsom** [comment]
A design idea: if I own an anthology, it could have two tabs on the index: one for documents, one for students. @timdunn22 how long would you estimate to implement that?

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -16,6 +16,64 @@ def show
     gon.rabl template: 'app/views/users/show.rabl', as: 'user'
   end
 
+  def remove_user
```
Comment (jamiefolsom) [inline app/controllers/users_controller.rb]: Should this function be called `remove_anthology`? Not a blocker.

File: app/controllers/documents_controller.rb [hunk 2]
```diff
@@ -67,9 +114,9 @@ def index
   # GET /documents/1
   # GET /documents/1.json
   def show
-    if request.path != document_path(@document)
-      redirect_to @document, status: :moved_permanently
-    end
+    # if request.path != document_path(@document)
```
Comment (jamiefolsom) [inline app/controllers/documents_controller.rb]: In general, unless there's a strong reason not to, let's remove commented out code. Not a blocker.

File: app/controllers/anthologies_controller.rb [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        format.json { render json: @anthology, status: :created, location: @anthology }
+      else
+        format.html { render action: "new" }
+        format.json { render json: @anthology.errors, status: :unprocessable_entity }
+      end
+    end
+
+  end
+
+  def remove_doc
```
Comment (jamiefolsom) [inline app/controllers/anthologies_controller.rb]: Why is there a `remove_doc` and not a `remove_user`?

File: config/routes.rb [hunk 4]
```diff
@@ -16,6 +19,10 @@
   get 'documents/catalog', to: 'catalog#index'
   get 'documents/catalog/image/:eid', to: 'catalog#image'
   get 'documents/catalog/reference/:eid', to: 'catalog#reference'
+  post 'anthology_add', to: 'documents#anthology_add'
+  post 'user_anthology_add', to: 'users#anthology_add'
+  post 'remove_user', to: 'users#remove_user'
+  post 'remove_doc', to: 'anthologies#remove_doc'
```
Comment (jamiefolsom) [inline config/routes.rb]: Routing and controller semantics are a little messy here, but let's go with it and clean up later if/as necessary

**Pull request merged**